component/libfuse/libfuse.go

```diff
@@ -304,6 +317,10 @@ func (lf *Libfuse) Configure(_ bool) error {
 	log.Info("Libfuse::Configure : read-only %t, allow-other %t, allow-root %t, default-perm %d, entry-timeout %d, attr-time %d, negative-timeout %d, ignore-open-flags %t, nonempty %t, direct_io %t, max-fuse-threads %d, fuse-trace %t, extension %s, disable-writeback-cache %t, dirPermission %v, mountPath %v, umask %v",
 		lf.readOnly, lf.allowOther, lf.allowRoot, lf.filePermission, lf.entryExpiration, lf.attributeExpiration, lf.negativeTimeout, lf.ignoreOpenFlags, lf.nonEmptyMount, lf.directIO, lf.maxFuseThreads, lf.traceEnable, lf.extensionPath, lf.disableWritebackCache, lf.dirPermission, lf.mountPath, lf.umask)
 
+	if common.GenConfig {
```

Comment:
What's the significance of doing this?

Reply:
so that we don't call genconfig func in normal flow of blobfuse but only while using gen-config command. I guess it wouldn't matter much since we are only writing to the config file in gen-config but yeah that's why its there